README.md

```diff
@@ -1 +1,128 @@
-# Cedrichs-Portfolio
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Cedrich's-Portfolio] <a name="about-project"></a>
+
+**[Cedrich's-Portfolio]** is a portifolio for Cedrich Mandeya and it contains information of his professional abilities.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+- A Browser and an internet connection.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Use commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:ceddddy/Cedrichs-Portfolio.git
+```
+
+### Install
+
+Install this project with:
+
+- Once you clone it into your desired folder you will be able to run it on your browser.
+
+### Usage
+
+To run the project, do the following:
+
+- Double click the html file or use liveserver if you choose to run it via vs-code
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Cedrich**
+
+- GitHub: [@githubhandle](https://github.com/ceddddy)
+- Twitter: [@twitterhandle](https://twitter.com/CedyDev)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/cedrich-mandeya-4118a91a7/)
+
```

Comment:
- [ ] Kindly make sure you also replace the template handles with yours like @ceddddy  instead of @githubhundle. :recycle: